demos/background_subtraction_demo/python/models.lst

```diff
@@ -1,3 +1,5 @@
 # This file can be used with the --list option of the model downloader.
 # TODO: instance-segmentation-person-????
 yolact-resnet50-fpn-pytorch
+# TODO: background-matting-mobilenetv2
+# TODO: robust-video-matting
```

Comment:
Please also add comments to tests

Reply:
Tests updated with commented bgr-matting models.